kornia/utils/helpers.py

```diff
@@ -252,16 +248,6 @@ def safe_inverse_with_mask(A: Tensor) -> Tuple[Tensor, Tensor]:
     else:
         dtype = dtype_original
 
-    if TYPE_CHECKING:
-        # TODO: remove this branch when kornia relies on torch >= 1.9
-        dummy: Tensor = torch.tensor(0)
-        return dummy, dummy
-    else:
-        if torch_version_geq(1, 9):
-            from torch.linalg import inv_ex  # (not available in 1.8)
-
-            inverse, info = inv_ex(A.to(dtype))
-            mask = info == 0
-            return inverse.to(dtype_original), mask
-        else:
-            NotImplementedError('This function just work with pytorch newer than 1.9.0')
+    inverse, info = inv_ex(A.to(dtype))
+    mask = info == 0
+    return inverse.to(dtype_original), mask
```

Comment:
@johnnv1 seems that this functions is not covered by any test. Is it actually used ? /cc @ducha-aiki

Reply:
Seems to be used just here https://github.com/kornia/kornia/blob/74e845bb4934355b4c6f134612e5f868fb03f5e7/kornia/geometry/homography.py#L75

Reply:
This is covered by tests but not for typing, we can see that on `codecov` site, where show negative cover for the typing flags

![image](https://user-images.githubusercontent.com/20444345/203566124-6348714d-3ff2-4fcb-b6fb-ea38fbe29eea.png)

Reply:
is there anything we can do to prevent showing in the PR checks ? this might block the `auto-merge` feature all the time which comes very handy

Reply:
I don't know if it would be possible to have some specific configuration to ignore some flag -- personally, I would just remove the typing coverage from codecov (will also solve this https://github.com/kornia/kornia/pull/2009#issuecomment-1321121618)

Reply:
Perfect